test/OpenTelemetry.Tests/Trace/Propagation/TraceContextPropagatorTest.cs

```diff
@@ -193,10 +193,10 @@ public void Inject_WithTracestate()
     public void DuplicateKeys()
     {
         // test_tracestate_duplicated_keys
-        Assert.Empty(CallTraceContextPropagator("foo=1,foo=1"));
-        Assert.Empty(CallTraceContextPropagator("foo=1,foo=2"));
-        Assert.Empty(CallTraceContextPropagator(new[] { "foo=1", "foo=1" }));
-        Assert.Empty(CallTraceContextPropagator(new[] { "foo=1", "foo=2" }));
+        Assert.Equal("foo=1,foo=1", CallTraceContextPropagator("foo=1,foo=1"));
+        Assert.Equal("foo=1,foo=2", CallTraceContextPropagator("foo=1,foo=2"));
```

Comment:
Is the correct behavior or is the behavior in @pellared's PR for opentelemetry-go correct? That is:

```csharp
Assert.Equal("foo=1,foo=1", CallTraceContextPropagator("foo=1"));
Assert.Equal("foo=1,foo=2", CallTraceContextPropagator("foo=1"));
```

Reply:
Ok, nevermind. After speaking with @utpilla, sounds like both behaviors are correct and left up to the implementation.